tests/common/mod.rs

```diff
@@ -479,6 +479,20 @@ fn bytes() {
     assert_eq!(deserialized.as_ref(), orig_src);
 }
 
+#[wasm_bindgen_test]
+fn bytes_as_array() {
+    let src = [1, 2, 3];
```

Comment:
One more thing that would be useful here - but I realised only after merging - is to actually have a test that shows that deserializing an array with non-byte values fails (e.g. `[1000, "42"]`).

This might be useful as a regression test in case someone tries to optimise this in future e.g. by doing `Uint8Array::new(...)` like in your original comment.

@bhalleycf Any chance you want to make a small follow-up PR with such regression test?

Reply:
I can do that, it's a good idea.

Reply:
Thanks for making a release so quickly too!